Eastern-Research-Group/mywaterway
# Feature/add highlights to content

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3672382

## Main Changes:
* Added highlights to most accordion content. This is to be consistent with the way the protected areas content is highlighted.
* Added protected areas content. 
* Added View On Map buttons to protected areas and wild and scenic rivers content.

## Steps To Test:
1. Navigate to http://localhost:3000/community/180101010308/protect
2. Verify the Protected Areas content displays and works correctly
3. Test the view on map button for the Protected Areas
4. Test the view on map button for the Wild and Scenic Rivers layer
5. Go to the monitoring tab and click the view on map button
6. Verify the accordion content is highlight, much like the content for the protected areas items.
